vagrant/README.md

```diff
@@ -0,0 +1,79 @@
+# About
+
+This is a fairly minimal Vagrantfile and associated bootstrap scripts for
```

Comment:
Would it good to add a note that we are only supporting the `GNUARM` compiler for vagrant setup?

Reply:
Done